packages/ui/src/domainModules/collectionMammals/components/CatalogNumberModal/index.js

```diff
@@ -1,10 +1,10 @@
 import React, { PureComponent } from 'react'
 import { withRouter } from 'react-router-dom'
 import PropTypes from 'prop-types'
+import { Button, Grid, Modal } from 'semantic-ui-react'
```

Comment:
this commit should be fixuped with https://github.com/DINA-Web/dina-collections/pull/283/commits/54322b2a4713a09eb4a8103609921c12773fb2b0 because you are reverting changes you did in that commit. maybe adding FormModal could be another commit before that one